src/renderer/prefComponents/common/range/index.vue

```diff
@@ -65,8 +65,14 @@ export default {
   margin: 20px 0;
   font-size: 14px;
   color: var(--editorColor);
+  width: 100%;
+  & .value {
+    text-align: right;
+    font-style: italic;
+    float: right
+  }
   & .el-slider {
-    width: 300px;
+    width: 100%;
```

Comment:
I prefer the old style, and not set the witch to `100%`, @fxha what's your opinions?

Reply:
I think full width with component grouping would be a vast UX improvement.